_training/curriculum.md

```diff
@@ -45,7 +39,11 @@ The curriculum is comprised of a set of standardized *modules*, so that students
 
 {% include list_of_selected_training_modules.html ids="uproot,grid,parallel,alpaka,yadagereana,doc,generators" %}
 
-## Further reads
+## <a name="bkginfo"></a>Background information and further reads
+
+Training in software and computing are essential ingredients for the success of any HEP experiment. As most experiments have similar basic prerequisites (Unix shell, Python, C++, …) we want to join our efforts and create one introductory software training curriculum that serves HEP newcomers the software skills needed as they enter the field, and in parallel, instill best practices for writing software.
+
+The curriculum is comprised of a set of *modules* (developed by HSF, Software Carpentries, and other organization), so that learners students can focus on what is most relevant to them.
```

Comment:
```suggestion
The curriculum is composed of a set of *modules* (developed by HSF, Software Carpentries, and other organizations), so that learners/students can focus on what is most relevant to them.
```